tests/encfs/primary.Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:latest
+WORKDIR /usr/src/app
+
+RUN pip install flask
+
+COPY primary.py .
+
+CMD ["python3", "primary.py"]
```

Comment:
Nit: can this be just python like it is in the examples repo?

Reply:
Doesn't make a difference to me. Sometimes specifying the version (or not specifying it) can cause errors in the container if it's not aliased properly. So as long as it works, it's fine as is.